source/geometry_model/interface.cc

```diff
@@ -182,7 +182,8 @@ namespace aspect
             errno = 0;
```

Comment:
Would it make sense to use `Utilities::string_to_int` here since that now hopefully should have the error checking?